[Red Glitchless] Add "intermediate route"

SquareSheepofRoot opened the pull request:
Adding intermediate red route. Includes a pastebin linked to simplified backup boom strats - ideally this could become a document under resources in the future.

SquareSheepofRoot committed:
Create README.md

Adding intermediate red route. Includes a pastebin linked to simplified backup boom strats - ideally this could become a document under resources in the future.

SquareSheepofRoot committed:
Update README.md

SquareSheepofRoot commented:
Just made an edit to weedle guy fight & several formatting edits.

SquareSheepofRoot committed:
Update README.md

Missed one thing.

SquareSheepofRoot committed:
Update README.md

Fixing unintended formatting. Hopefully good now.

SquareSheepofRoot committed:
Update README.md

SquareSheepofRoot committed:
Update README.md

SquareSheepofRoot committed:
Update README.md

hwangbro commented on docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md:
```diff
@@ -0,0 +1,714 @@
+## MT. MOON SPLIT
+
+**Moon Manip Save**
+- 1- 8
```
Thoughts on simplifying this to "heal to 36+"

SquareSheepofRoot replied:
Yes, did this and added an if confident with yolo ball under 9 hp then delay heal clause.

hwangbro commented on docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md:
```diff
@@ -0,0 +1,714 @@
+
+Do Route 3 Moon Manip https://pastebin.com/tggXpQRC
+
+>The Lass that you see on screen before entering Mt. Moon during the manip should be stuck facing up; however on Frame 36 & Frame 37 she walks right & faces left, respectively. If you'd like to learn the alternative movement used to manipulate mt. moon on these frames check here (../../../resources/route-3-moon-backups.md)
```
Can you make the f36/37 reference a hyperlink?

SquareSheepofRoot replied:
Yes fixed this link.

hwangbro commented on docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md:
```diff
@@ -0,0 +1,714 @@
+  + Use Moon Stone
+  + Teach Mega Punch (TM01) in Slot 1
+
+> Moon Exp: KO a Geodude or Paras if you happen to encounter one while walking up to talk to the Nerd
```
Maybe superfluous, mention that you use PS to kill Paras?

SquareSheepofRoot replied:
Yup, makes sense. Added WG geodude as well cause why not

hwangbro commented on docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md:
```diff
@@ -0,0 +1,714 @@
+- X Acc + X Speed + Blizzard + (TB)
+- HD x5
+
+Pick up the hidden Max Ether
```
Should link to pokeworld here

SquareSheepofRoot replied:
Yes added it.

hwangbro commented on docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md:
```diff
@@ -0,0 +1,714 @@
++ **(126+)** Don't Heal
+
+Lance:
+- X Special + TB (if hydro pump misses you can stall with EQ or with an X-Acc if you still have 2 left )
```
Maybe mention x speed stall hps here as well

SquareSheepofRoot replied:
Yes, HP < 133 right?

SquareSheepofRoot committed:
Update README.md

Simplified Moon Manip Save Healing, Added F36/F37 Link, Added WG Geo, PS Paras Moon EXP; Link for Max Ether; X-speed stall if HP<133.
